src/main/java/com/sureit/admin/attendance/service/EmployeeService.java

```diff
@@ -10,24 +8,24 @@
 import org.springframework.stereotype.Service;
 
 import com.sureit.admin.attendance.model.Employee;
-import com.sureit.admin.attendance.model.EmployeeLogin;
+import com.sureit.admin.attendance.model.TimeSheetEntry;
 import com.sureit.admin.attendance.repository.IEmployeeDAO;
-import com.sureit.admin.attendance.repository.IEmployeeLoginDAO;
+import com.sureit.admin.attendance.repository.ITimeSheetEntryDAO;
 
 @Service("employeeService")
 public class EmployeeService {
 
 	@Autowired
 	private IEmployeeDAO employeeDAO;
-	
+
 	@Autowired
-	private IEmployeeLoginDAO employeeLoginDAO;
+	private ITimeSheetEntryDAO timeSheetEntryDAO;
 
 	public List<Employee> getAllEmployees() {
 		List<Employee> employeeList = null;
 		try {
 			employeeList = (List<Employee>) employeeDAO.findAll();

```

Comment:
Check what is return value of findAll(); if it returns null in case there are no record found then you can return this result from here instead storing it in employeeList variable and returning it later,